lib/utils/result_formatter.test.ts

```
@@ -95,3 +95,57 @@ describe('ResultFormatter timestamp family NaN guards', () => {
     expect(dr.formatted.items[0].value).toBe('02:08:00');
   });
 });
+
+describe('ResultFormatter numeric formatters NaN guards', () => {
+  // Non-numeric ACARS fields ('---', '***', whitespace) become NaN via
+  // Number()/parseInt(); these must disappear from formatted.items rather
+  // than surfacing as literal "NaN" strings (issue #494).
+  const cases: Array<[string, (dr: DecodeResult) => void, string]> = [
+    ['heading', (dr) => ResultFormatter.heading(dr, NaN), 'heading'],
+    ['groundspeed', (dr) => ResultFormatter.groundspeed(dr, NaN), 'groundspeed'],
```

ESLint

[error] Replace `'groundspeed',·(dr)·=>·ResultFormatter.groundspeed(dr,·NaN),·'groundspeed'` with `⏎······'groundspeed',⏎······(dr)·=>·ResultFormatter.groundspeed(dr,·NaN),⏎······'groundspeed',⏎····`

(prettier/prettier)